front/app/containers/App/constants.js

```diff
@@ -86,6 +87,7 @@ exports.appLocalePairs = {
   'ro-RO': 'Română',
   'pt-BR': 'Português (Brasil)',
   'ar-SA': 'عربى',
+  'ar-MA': 'اللهجة المغربية (Morocco)',
```

Comment:
I think this is wrong. 'Morocco' needs to be in the local language too. See languages above.

Reply:
@brentguf Change to... ? Or remove?

Reply:
@jinjagit I would again trust Google Translate and Wiki here. So (Morocco) should be (المغرب).

Reply:
If you check the other languages, you could probs figure it out 🙂 We should leave out the country name. It's only there if it's not clear from the language name. Eg Dutch (Belgium) and Dutch (Netherlands). For Romanian, it just says that because there's only 1 variant.